aws-ecs-byovpc/example.tfvars

```diff
@@ -0,0 +1,14 @@
+nuon_id = "28g232g0an6vh29t6mu7kv3a96"
+region  = "us-east-2"
+# assume_role_arn  = "arn:aws:iam::949309607565:role/nuon-demo-install-access"
+assume_role_arn  = "" # This needs to be set for use from our services but should not be when running locally.
+install_role_arn = "arn:aws:iam::618886478608:role/install-k8s-admin-stage"
```

Comment:
It looks like some EKS stuff got left here.

Reply:
Yep, cleaning up. Thanks!